packages/cli/src/commands/generate/service/__tests__/fixtures/multiWord_crud.ts

```diff
@@ -1,22 +1,18 @@
-import {
-  UserProfileWhereUniqueInput,
-  UserProfileCreateInput,
-  UserProfileUpdateInput,
-} from '@prisma/client'
+import { Prisma } from '@prisma/client'
```

Comment:
Should this be the default export, so like:
```
import Prisma from '@prisma/client'
```

Question, not a suggestion :)

Reply:
@dac09 The Prisma docs specify `import { Prisma } from '@prisma/client'`: https://www.prisma.io/docs/concepts/components/prisma-client/working-with-generated-types

Also the Prisma code mod updates all of the type imports to this syntax as well:
https://github.com/prisma/codemods

I just tested in my codebase and the Prisma namespace is only available if you were to do something like `Prisma.Prisma.ModelWhereUniqueInput`.